src/lib/dnssd/Discovery_ImplPlatform.cpp

```diff
@@ -746,6 +765,31 @@ CHIP_ERROR DiscoveryImplPlatform::DiscoverCommissioners(DiscoveryFilter filter,
     return error;
 }
 
+CHIP_ERROR DiscoveryImplPlatform::DiscoverOperational(DiscoveryFilter filter, DiscoveryContext & context)
+{
+    ReturnErrorOnFailure(InitImpl());
+    StopDiscovery(context);
+
+    char serviceName[kMaxCommissionerServiceNameSize];
```

Comment:
Why is kMaxCommissionerServiceNameSize the right thing here?  That doesn't seem related to operational anything.

Reply:
You are right. I will fix this.

Reply:
fixed the length to use kMaxOperationalServiceNameSize.